pkg/netpol/scan/error_types.go

```diff
@@ -151,6 +165,14 @@ func noK8sResourcesFound() *FileProcessingError {
 	return &FileProcessingError{&NoK8sResourcesFoundError{}, "", 0, -1, false, false}
 }
 
+func noK8sWorkloadResourcesFound() *FileProcessingError {
+	return &FileProcessingError{&NoK8sWorkloadResourcesFoundError{}, "", 0, -1, false, false}
+}
+
+func noK8sNetworkPolicyResourcesFound() *FileProcessingError {
+	return &FileProcessingError{&NoK8sNetworkPolicyResourcesFoundError{}, "", 0, -1, false, false}
+}
+
```

Comment:
Shouldn't these two be severe errors?

Reply:
ok, will change `NoK8sResourcesFoundError` , `NoK8sWorkloadResourcesFoundError` , `NoK8sNetworkPolicyResourcesFoundError` to severe errors instead of warnings.